### PR TITLE
Lpr 28 contribution

### PR DESCRIPTION
**Pull request by NealGeilen**
Added a README and pull request template
#59 needs to be accepted first

### CODE REVIEW

**Review by nickonos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Gettings started
+
+## Prerequisites
+* Git
+* Docker (version 19.03.0+)
+* Docker compose (Comes pre-installed with Docker Desktop)
+## Steps
+1. Clone this repository, using the following command: ``git clone https://github.com/looplanguage/api.pkg.looplang.org``
+2. Browse to the folder of the repository
```
Comment (WouterPennings) [inline README.md:25]: Maybe something like:
2. ``cd api.pkg.looplang.org``

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+This repository contains the backend logic and API for the package manager. 
+
+# Gettings started
+
+## Prerequisites
+* Git
+* Docker (version 19.03.0+)
+* Docker compose (Comes pre-installed with Docker Desktop)
+## Steps
+1. Clone this repository, using the following command: ``git clone https://github.com/looplanguage/api.pkg.looplang.org``
```
Comment (WouterPennings) [inline README.md:24]: 0. Open Terminal?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<hr/>
+
+The Loop language is a new programming language for data science and AI. Loop uses a package manager to manage the various packages that users and companies are developing for the language. 
+
+This repository contains the backend logic and API for the package manager. 
+
+# Gettings started
+
+## Prerequisites
```
Comment (WouterPennings) [inline README.md:19]: I assume this is for windows, is it the same for linux and macos?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Git
+* Docker (version 19.03.0+)
+* Docker compose (Comes pre-installed with Docker Desktop)
+## Steps
+1. Clone this repository, using the following command: ``git clone https://github.com/looplanguage/api.pkg.looplang.org``
+2. Browse to the folder of the repository
+3. Run the command: ``docker-compose up``
```
Comment (WouterPennings) [inline README.md:26]: image
 I get this error....

**Review by WouterPennings — CHANGES_REQUESTED** [review]

**Review by WouterPennings — APPROVED** [review]

**Pull request merged**